packages/rum-core/package.json

```diff
@@ -13,15 +15,21 @@
     "directory": "packages/rum-core"
   },
   "scripts": {
+    "prepublish": "npm run build:main",
+    "build:main": "npx babel src -d lib --config-file ../../babel.node.js",
```

Comment:
Maybe we should put this under `dist/lib` as well, WDYT?

Reply:
sure, lets do it.